provision/PROVISION.md

```diff
@@ -0,0 +1,30 @@
+## Provision
+
+Note: This provisioning model only supports Keycloak Legacy distribution, we are working on a version to support Quarkus based distribution.
+
+#### Keycloak
+A light weight keycloak server setup with, a keycloak container and another postgres container as its external store.
+
+ - The image used for keycloak server is the latest:legacy container from [quay.io](https://quay.io/repository/keycloak/keycloak?tab=tags&tag=legacy)
+ - To control the cpu quota for the containers, we use cpusets for both containers (keycloak and postgres), that can be changed from the [keycloak-postgres.yml](keycloak/keycloak-postgres.yml)
+
+#### Bringing up the containers
+
+To run in a detached mode,
+
+`docker-compose -f keycloak-postgres.yml up -d`
```

Comment:
We're using the `latest` image of Keycloak. It might be helpful to ask people to now and then run `docker-compose pull` to fetch the latest versions of the images.

Reply:
sure, I can add a note section underneath this and its a good idea :) thanks for the suggestion.